pkg/iss/leaderselectionpolicy/blacklist.go

```diff
@@ -28,15 +28,16 @@ func NewBlackListLeaderPolicy(members []t.NodeID, MinLeaders int) *BlacklistLead
 	}
 }
 
-// Leaders always returns the whole membership for the SimpleLeaderPolicy. All nodes are always leaders.
+// Leaders returns a list of least recently suspected leaders
```

Comment:
Typo: return

Reply:
I see that it sounds a bit odd, but `Leaders` is a name of the function that returns a list.